observability/dynatrace/dynakube.yaml

```diff
@@ -8,12 +8,14 @@ metadata:
 spec:
   apiUrl: https://ypd98635.sprint.dynatracelabs.com/api
   oneAgent:
-    applicationMonitoring:
-      useCSIDriver: true
+    cloudNativeFullStack:
```

Comment:
@vfarcic we would need to change this in the composition as well to enable log monitoring